geoengine/layers.py

```diff
@@ -394,6 +394,11 @@ def add_existing_collection(self,
 
         return collection_id
 
+    def get_items_by_name(self, name: str) -> List[Listing]:
+        '''Get all children with the given name'''
+
+        return [item for item in self.items if item.name == name]
```

Comment:
Can there by more than one item with that name?
Could there be a wildcard, like `'name.*"?

And…could you then update https://github.com/geo-engine/geoengine-python/blob/main/examples/layers.ipynb?

Reply:
i can add 2 layers with the same name so i guess that it is possible. I had no need for a wildcard yet. So i would suggest we add that when it is needed.

Reply:
[d3bc79b](https://github.com/geo-engine/geoengine-python/pull/118/commits/d3bc79b55b3276c19f7dcde240bce3cf8757465f)